syntax/blog.php

```diff
@@ -121,9 +123,9 @@ function render($mode, Doku_Renderer $renderer, $data) {
         // now include the blog entries
         foreach ($entries as $entry) {
             if ($mode == 'xhtml' || $mode == 'code') {
-                if(auth_quickaclcheck($entry['id']) >= AUTH_READ) {
+                if(isset($entry['id']) && (auth_quickaclcheck($entry['id']) >= AUTH_READ)) {
```

Comment:
Did you also got a warning about id was not set? Or is this about another issue? Then I expect more functions before have already problems..

Reply:
Not sure what you mean? There is another patch in line 128 that also involves `$entries['id']` but there the value is not found in `$this->included_pages` so I added a check.

This PR fixed all of the warnings I saw.

Reply:
Oh, and you can see the PHP warnings I was fixing in the comments for the individual commits that make up this PR.

Reply:
Ah, sorry! Nice that you add the actual warnings, that is helpful.

Could it be that the error which refers to line 126, is the same error as the one which refers to line 128, but becomes 128 after the fix of the undefined ‘first’ key. That change adds two extra lines. 
`$entry['id']` should always be defined, of course an extra check on line 126 does not break anything, but it suggests also it can be undefined, what is not possible so far I understood the code. So my preference is to do not that isset check there.

Reply:
Testing a single load of my main Blog page with `&purge`:
Before the patch the first warning (L126) happened a few times. Then later at the end of the list of warnings in the log the second warnings (L128) popped up for a bunch of page ids. So I think they are separate issues. Otherwise I would have expected them to alternate.

The first concerns the `'id'` element of `$entry` missing or having a `null` value.
The second concerns the `$entry['id']` element of `$this->included_pages` missing. And in these cases `$entry['id']` has plausible values.
So again unrelated I think.

If you think the first case should never happen then there is another bug lurking somewhere.